CHANGELOG.md

```diff
@@ -40,6 +40,9 @@ _Code:_
 
   - Updated Objective-C examples (iOS and macOS, Carthage and CocoaPods) to showcase usage of the newest Secure Cell API: generating symmetric keys and using Secure Cell with Passphrase ([#688](https://github.com/cossacklabs/themis/pull/688)).
   
+- **Rust**
+
+  - Dropped `libthemis-src` crate support. From now the feature `vendored` won't be available for RustThemis wrapper and `libthemis-sys` will only link with Themis libraries, and not build them by itself.
```

Comment:
Here's a cleaner version:

```suggestion
  - Dropped `libthemis-src` crate support and removed the `vendored` feature. RustThemis wrapper now requires Themis Core to be installed in the system ([#691](https://github.com/cossacklabs/themis/pull/691)).
```

@iamnotacake, please mention PRs in the changelog, that makes easier to follow the changes. You can typically predict the PR number before submitting it, or you can always edit it later, once your PR has been assigned a number.